flow/layers/picture_layer.cc

```diff
@@ -26,9 +26,6 @@ void PictureLayer::Preroll(PrerollContext* context, const SkMatrix& matrix) {
 
     SkMatrix ctm = matrix;
     ctm.postTranslate(offset_.x(), offset_.y());
```

Comment:
Should this offset matter any more? It will matter during draw, but the translation parts of the matrix are ignored now.

Reply:
You're right. Removed it.